### PR TITLE
Expose IdentityId in cognito_credentials to upload to S3 user folder

### PR DESCRIPTION
**Pull request by BenV**
Merged in changes from https://github.com/jonsaw/amazon-cognito-identity-dart/pull/55 to expose and document the user identity ID

### CODE REVIEW

File: lib/src/cognito_credentials.dart [hunk 1]
```diff
@@ -30,14 +32,14 @@ class CognitoCredentials {
     if (expireTime == null ||
         DateTime.now().millisecondsSinceEpoch > expireTime - 60000) {
       final identityId = CognitoIdentityId(_identityPoolId, _pool);
-      final identityIdId = await identityId.getIdentityId(token, authenticator);
```
Comment (furaiev) [inline lib/src/cognito_credentials.dart]: Hi, thanks for contribution,
Could you please explain why did you remove `authenticator` from getIdentityId method? It's optionally used there.

Comment (BenV) [reply lib/src/cognito_credentials.dart]: Ah, there was a merge conflict and I probably just missed it. Will get that resolved.

Comment (BenV) [reply lib/src/cognito_credentials.dart]: @furaiev Should be good to go now, sorry about that.

Comment (furaiev) [reply lib/src/cognito_credentials.dart]: It's ok, thank you! :)

**Comment by furaiev** [comment]
Thank you for contribution!
Available as of v0.1.9
